sources/editor/Stride.GameStudio/View/Theme.AvalonDock.xaml

```diff
@@ -0,0 +1,1398 @@
+<!--***********************************************************************************
+
+   Extended WPF Toolkit
+
+   Copyright (C) 2007-2013 Xceed Software Inc.
+
+   This program is provided to you under the terms of the Microsoft Public
+   License (Ms-PL) as published at http://wpftoolkit.codeplex.com/license
+
+   For more features, controls, and fast professional support,
+   pick up the Plus Edition at http://xceed.com/wpf_toolkit
+
+   Stay informed: follow @datagrid on Twitter or Like http://facebook.com/datagrids
+
+  **********************************************************************************-->
+
+<ResourceDictionary
```

Comment:
Why do we need this file, where does it come from ?

Reply:
Oh... I think I copied by mistake, I undid the changes.